build.gradle

```diff
@@ -6,6 +6,8 @@ apply plugin: 'jacoco'
 group = "com.stepstone.inbucket"
 version = "1.0.1-SNAPSHOT"
 
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
```

Comment:
What is the point in making artifact incompatible with 1.7, when we don't use any features introduced in Java 1.8?
It is build with java 1.8, but could be used on systems with java 1.7